packages/gatsby-plugin-utils/src/polyfill-remote-file/graphql/gatsby-image-resolver.ts

```diff
@@ -241,6 +241,12 @@ export async function gatsbyImageResolver(
     }
   }
 
+  if (
+    [`true`, `1`].includes(process.env.GATSBY_SHOULD_TRACK_IMAGE_CDN_URLS || ``)
```

Comment:
This should be memoized / calculated once. `process.env` is slow and doing this on each image will add that penalty to all "images" ( https://github.com/nodejs/node/issues/3104 )

Reply:
Fixed